draft-ietf-taps-interface.md

```diff
@@ -1139,8 +1139,8 @@ Default:
 : Ignore
 
 This property specifies whether an application considers it useful to be
-informed in case sent data was retransmitted more often than a certain
-threshold (see {{conn-excss-retransmit}} for configuration of this threshold).
+informed in case sent data was lost more often than a certain
+threshold (see {{conn-excss-loss}} for configuration of this threshold).
```

Comment:
Okay… I agree on that one – In case we want to keep the semantics of this PR (see comment below) packets of the PR, it hast to say 
```suggestion
informed in case packets of the underlying transport protocol have been lost more often than a certain
threshold (see {{conn-excss-loss}} for configuration of this threshold).
```
or it makes no sense.